### PR TITLE
add challenge data  CoursesPage.js, ChallengePage.js 

### PR DESCRIPTION
**Pull request by kalianchuk**
deleted Courses.js
moved CoursesPage.js class  to _PageObject
moved ChallengePage.js  class  to _PageObject

### CODE REVIEW

File: modules/_PageObjects/ChallengePage.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
+import AppPage from "../AppPage";
```
Comment (Vladyslavva) [inline modules/_PageObjects/ChallengePage.js]: Change, please, to single quotes

File: modules/_PageObjects/CoursesPage.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import AppPage from "../AppPage";
+import LoginPage from "../_PageObjects/LoginPage";
+import {admin} from "../user/login/loginRole_data";
```
Comment (Vladyslavva) [inline modules/_PageObjects/CoursesPage.js]: Single quotes in all imports. Change, please

File: modules/courses/courses_challenge/coursesChallenge.spec.js [hunk 3]
```diff
@@ -1,19 +1,32 @@
 import LoginPage from "../../_PageObjects/LoginPage";
 import {admin} from "../../user/login/loginRole_data";
```
Comment (Vladyslavva) [inline modules/courses/courses_challenge/coursesChallenge.spec.js]: Same here with quotes

File: modules/courses/courses_challenge/coursesChallenge.spec.js [hunk 4]
```diff
@@ -1,19 +1,32 @@
...
-import Courses from "../Courses";
-import Challenge from "./Challenge";
-import {course} from "../Courses";
-import {challenge} from "./Challenge";
+import CoursesPage from "../../_PageObjects/CoursesPage";
+import ChallengePage from "../../_PageObjects/ChallengePage";
+import {course} from "../../_PageObjects/CoursesPage";
+import {challenge} from "../../_PageObjects/ChallengePage";
```
Comment (Vladyslavva) [inline modules/courses/courses_challenge/coursesChallenge.spec.js]: Same

File: modules/_PageObjects/ChallengePage.js (hunk 1 above)
Comment (Vladyslavva) [reply modules/_PageObjects/ChallengePage.js]: And Katya moved your file Challenge.js to PageObjects, check, please

**Comment by kalianchuk** [comment]
didn't figure out how to resolve conflict 
created and merged  new pull request

**Pull request closed without merging**